src/com/team949/Robot.java

```diff
@@ -105,133 +107,60 @@ public void disabledPeriodic() {
 	 */
 	@Override
 	public void autonomousInit() {
-//		char startingPosition = startingPositionChooser.getSelected();
-//		String targetScoring = targetScoringChooser.getSelected();
-//		String gameData = DriverStation.getInstance().getGameSpecificMessage();
-		//autonomousCommand = autonomousSwitchLogic(startingPosition, targetScoring, gameData);
-		
-//		String autoSelected = SmartDashboard.getString("Auto Selector", "Default"); 
-//		switch(autoSelected) {
-//		case "My Auto": autonomousCommand = new MyAutoCommand();
-//		break; 
-//		case "Default Auto": 
-//		default:
-//		autonomousCommand = new ExampleCommand(); break; }
+		// char startingPosition = startingPositionChooser.getSelected();
+		// String targetScoring = targetScoringChooser.getSelected();
+		// String gameData =
+		// DriverStation.getInstance().getGameSpecificMessage();
+		// autonomousCommand = autonomousSwitchLogic(startingPosition,
+		// targetScoring, gameData);
+
+		// String autoSelected = SmartDashboard.getString("Auto Selector",
+		// "Default");
+		// switch(autoSelected) {
+		// case "My Auto": autonomousCommand = new MyAutoCommand();
+		// break;
+		// case "Default Auto":
+		// default:
+		// autonomousCommand = new ExampleCommand(); break; }
```

Comment:
I'd rather not have large blocks of commented out code in the repository, unless we're planning to uncomment in the very near future. If we need to see old versions of the code, git can do that for us.